R/reference_quarto.R

```diff
@@ -0,0 +1,46 @@
+#' Convert .Rd files from man/ to .md files in docs/man/
+#' @param update If TRUE, overwrite existing files
+#' @keywords internal
+.make_reference_quarto <- function(update = FALSE) {
+
+  cli::cli_h1("Building reference")
+
+  # source and target file paths
+  # using here::here() breaks tests, so we rely on directory check higher up
+  man_source <- list.files(path = "man", pattern = "\\.Rd$")
+  man_target <- list.files(path = "docs/man", pattern = "\\.md$")
```

Comment:
Just in case you didn't see, docute and docsify use the folder `docs` while mkdocs uses `docs/docs` (unless something changed in the recent PRs), which is why I use `.doc_path()` across the code

Reply:
got it